client/app/scripts/utils/metric-utils.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+import d3 from 'd3';
+import { formatMetric } from './string-utils';
+import { colors } from './color-utils';
+
+
+const openFilesScale = d3.scale.log().domain([1, 100000]).range([0, 1]);
```

Comment:
openFilesScale will disappear w/ #1149.

Load will continue w/ a vague max. Current stuff in a metric object:

```js
{
  id: "process_cpu_usage_percent",
  label: "CPU",
  format: "percent",
  value: 4.16,
  samples: [],
  min: 0,
  max: 100,
  first: "2016-04-04T08:41:49.831200363Z",
  last: "2016-04-04T08:42:02.831776457Z"
}
```

`scale: "log"`, feels a bit funny in there... but i understand the gist... I'll make an issue.